tests/dataframe/test_groupby_pytest.py

```diff
@@ -224,15 +240,44 @@ def test_groupby_dataframe_mad(self):
         pd_flights = self.pd_flights().filter(self.filter_data + ["DestCountry"])
         oml_flights = self.oml_flights().filter(self.filter_data + ["DestCountry"])
 
-        pd_mad = pd_flights.groupby("DestCountry").mad()
+        pd_mad = pd_flights.groupby("DestCountry").apply(
+            lambda group: group.select_dtypes(include="number").apply(
+                lambda x: CustomFunctionDispatcher.apply_custom_function(
+                    MEAN_ABSOLUTE_DEVIATION, x
+                )
+            )
+        )
```

Comment:
Perhaps we can extract the lambda to a name so its more readable?

Im guessing what its doing (not proficient in pandas) 
```
groupByMeanAsbouluteDevaition =             lambda group: group.select_dtypes(include="number").apply(
                lambda x: CustomFunctionDispatcher.apply_custom_function(
                    MEAN_ABSOLUTE_DEVIATION, x
                )
            )
        )
```
 pd_mad = pd_flights.groupby("DestCountry").apply(groupByMeanAsbouluteDevaition)